util/compression.h

```diff
@@ -1048,7 +1048,6 @@ inline bool LZ4_Compress(const CompressionInfo& info,
 #else   // up to r123
   outlen = LZ4_compress_limitedOutput(input, &(*output)[output_header_len],
                                       static_cast<int>(length), compress_bound);
-  (void)ctx;
```

Comment:
Yes, this is a bug of rocksdb, `ctx` is not defined, on some server it will report a compile error (our build server, or Pegasus CI, I'm not sure now). And I've submmited this patch to rocksdb official repo.